services/node_update.proto

```diff
@@ -0,0 +1,75 @@
+syntax = "proto3";
+
+package proto;
+
+/*
+ * Hedera Network Services Protobuf
+ *
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+option java_package = "com.hederahashgraph.api.proto.java";
+// <<<pbj.java_package = "com.hedera.hapi.node.dynamic">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+
+import "google/protobuf/wrappers.proto";
+import "basic_types.proto";
+import "duration.proto";
+import "timestamp.proto";
+
+/**
+ * Modify the attribute of a node. If a field is not set in the transaction body, the
+ * corresponding node attribute will be unchanged.
+ * or phase 2, this marks node to be updated in the merkle tree and will be used to write config.txt and public.pfx during prepare freeze
+ * the node will not be updated until the network is upgraded.
+ * Original node account ID has to sign the transaction.
+ */
+message NodeUpdateTransactionBody {
+
+    /**
+     * The unique id of the Node to be updated. This must refer to an existing, non-deleted node.
+     */
+    uint64 node_id = 1;
```

Comment:
Our standard is to generally do `int64` for longs since Java doesn't support unsigned natively. Same comment everywhere. If we're changing that then we need to have a meeting to discuss the ramifications and agree to that going forward.

Reply:
Perhaps worth noting, the `node_id` is both uint64 and int64 in different places originally; uint64 seems the better choice to settle on to have all `node_id`
values the same type.

Reply:
I took @jsync-swirlds's suggestion and changed to uint64 from int64.